app/views/subscriptions/new.html.erb

```diff
@@ -50,7 +50,7 @@
         <input type="hidden" name="no_shipping" value="1">
 
         <!-- url to return to on success -->
-        <input type="hidden" name="return" value="<%= paypal_return_url %>">
+        <input type="hidden" name="return" value="<%= "#{paypal_return_url}?item_number=#{current_user.slug}&item_name=#{@plan.name}&auth=#{cookies['_WebsiteOne_session']}" %>">
```

Comment:
what would be awesome to keep the tests and this in sync, would be making this a helper method that we could also refer to in the test, that way we'd be sure the test was assembling the return_url the same way the form is